### PR TITLE
minor change to reducers, forgot to change = to : on initialstate

### PR DESCRIPTION
**Pull request by faulkenfett11**
# Description

Please include a summary of the change and a link to which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: react_django/react-src/package.json [hunk 1]
```diff
@@ -1,28 +1,26 @@
 {
-  "name": "react-src",
+  "name": "sidenavstuff",
```
Comment (LeonOfTheHollow) [inline react_django/react-src/package.json]: Why are we changing the name of the react app here?

File: react_django/react-src/src/components/HPprofile/hpProfileView.js [hunk 2]
```diff
@@ -1,86 +1,82 @@
...
-             </div>
-        )
-    }
-}
\ No newline at end of file
+import React, { Component } from 'react';
+import { FaEnvelope, FaCheck, FaLink, FaStar } from 'react-icons/fa'
+import './hpProfileView.css'
+
+ export default class StudentSettings extends Component{
```
Comment (LeonOfTheHollow) [inline react_django/react-src/src/components/HPprofile/hpProfileView.js]: Is this not the hiring partner info?

File: react_django/react-src/src/components/nav/job-seeker/JSnav.css [hunk 3]
```diff
@@ -1,4 +1,4 @@
-.jobseekerNav-container {
+.companyNav-container {
```
Comment (LeonOfTheHollow) [inline react_django/react-src/src/components/nav/job-seeker/JSnav.css]: Is this not the jobseeker navigation css?

File: react_django/django-src/django_app/models.py [hunk 4]
```diff
@@ -1,48 +1,42 @@
...
-
-    def __repr__(self):
-        return self.username
-
-    def __str__(self):
-        return self.username
+class City(models.Model):
+    id = models.AutoField(primary_key=True)
+    name = models.CharField(max_length=20, default='')
+    state = models.ForeignKey(State, on_delete=models.CASCADE, null=True)
```
Comment (20djm) [inline react_django/django-src/django_app/models.py]: probably don't need state in City - Other than that looks good!

**Pull request merged**